cbor/src/main/java/com/fasterxml/jackson/dataformat/cbor/CBORParser.java

```diff
@@ -817,7 +817,7 @@ protected String _numberToName(int ch, boolean neg) throws IOException
         if (neg) {
             i = -i - 1;
         }
-        return String.valueOf(1);
+        return String.valueOf(i);
```

Comment:
Fix checked in 2.7 (and actually even 2.6 although not sure there will be new releases from that branch)